sys/linux/fs_ioctl.txt

```diff
@@ -116,3 +119,13 @@ fsmap {
 fs_flags = FS_SECRM_FL, FS_UNRM_FL, FS_COMPR_FL, FS_SYNC_FL, FS_IMMUTABLE_FL, FS_APPEND_FL, FS_NODUMP_FL, FS_NOATIME_FL, FS_DIRTY_FL, FS_COMPRBLK_FL, FS_NOCOMP_FL, FS_ENCRYPT_FL, FS_BTREE_FL, FS_INDEX_FL, FS_IMAGIC_FL, FS_JOURNAL_DATA_FL, FS_NOTAIL_FL, FS_DIRSYNC_FL, FS_TOPDIR_FL, FS_HUGE_FILE_FL, FS_EXTENT_FL, FS_VERITY_FL, FS_EA_INODE_FL, FS_EOFBLOCKS_FL, FS_NOCOW_FL, FS_DAX_FL, FS_INLINE_DATA_FL, FS_PROJINHERIT_FL, FS_CASEFOLD_FL, FS_RESERVED_FL
 
 define FSMAP_SIZE	sizeof(struct fsmap)
+
+fsuuid2 {
+	len	int8
+	uuid	uuid_str
```

Comment:
I'm not sure it's of the `uuid_str` type.

https://github.com/google/syzkaller/blob/509da42949c4013fb236ebf6e25d3562d110198c/sys/linux/filesystem.txt#L253-L263

This is way longer than the 16 bytes here:

https://elixir.bootlin.com/linux/v6.11.6/source/include/uapi/linux/fs.h#L77